ouroboros-network/src/Ouroboros/Network/DeltaQ.hs

```diff
@@ -269,3 +275,21 @@ gsvRequestResponseDuration PeerGSV{outboundGSV, inboundGSV}
         gsvTrailingEdgeArrive outboundGSV reqSize
      <> gsvTrailingEdgeArrive inboundGSV respSize
 
+
+defaultGSV :: PeerGSV
+defaultGSV = PeerGSV { outboundGSV, inboundGSV }
+  where
+    default_g = 3600 -- 1h, start with an unreasonable large G.
+    default_s = 2e-6 -- 5Mbps.
```

Comment:
sorry to nit pick - but that is 4Mpbs

Reply:
I was lazy and stole it from https://github.com/input-output-hk/ouroboros-network/blob/6507ec6eb08e75925e9c3072a11db84e226bf706/ouroboros-network/src/Ouroboros/Network/BlockFetch.hs#L289